modules/ROOT/nav.adoc

```diff
@@ -35,53 +35,86 @@
-  *** xref:external-oauth-2.0-token-validation-policy.adoc[Mule OAuth 2.0 Access Token]
-  *** xref:oauth2-policies-new.adoc[OAuth 2 Policies concepts]
-  *** xref:oauth-policy-implementation-concept.adoc[OAuth 2 Policy Implementation]
-  *** xref:openam-oauth-token-enforcement-policy.adoc[OpenAM OAuth 2.0 Token Enforcement Policy]
-  *** xref:policy-openid-connect.adoc[OpenID Connect OAuth 2.0 Token Enforcement]
-  *** xref:policy-ping-federate.adoc[PingFederate OAuth 2.0 Token Enforcement]
-  *** xref:apply-oauth-token-policy-task.adoc[OAuth 2.0 Token Validation]
-  *** xref:rate-limiting-and-throttling.adoc[Rate Limiting and Throttling]
-  *** xref:configure-rate-limiting-task.adoc[Rate Limiting Policy v1.0.0 or v1.1.0]
-  *** xref:rate-limit-1.2.0-task.adoc[Rate Limiting Policy v1.2.0]
-  *** xref:rate-limiting-and-throttling-sla-based-policies.adoc[Rate Limiting and Throttling - SLA-Based Policies concepts]
-  *** xref:spike-control-reference.adoc[Spike Control]
-  *** xref:throttling-rate-limit-concept.adoc[Throttling and Rate Limiting]
-  *** xref:apply-configure-xml-threat-task.adoc[XML Threat Protection]
- ** xref:policies-custom-landing-page.adoc[Custom Policies]
-  *** xref:custom-policy-getting-started.adoc[Getting started with Custom Policies development]
-  *** xref:custom-policy-packaging-policy.adoc[Packaging a Custom Policy]
-  *** xref:custom-policy-uploading-to-exchange.adoc[Uploading a Custom Policy to Exchange]
-  *** xref:custom-policy-4-reference.adoc[Reviewing Custom Policy concepts]
-  *** xref:http-policy-transform.adoc[Reviewing HTTP Policy Transform Extension]
-  *** xref:add-remove-headers-concept.adoc[Adding/Removing headers Custom Policy example]
-   **** xref:add-remove-headers-latest-task.adoc[Adding/Removing Headers Custom Policy]
-   **** xref:add-remove-headers.adoc[Testing Adding/Removing headers Custom Policy example]
-  *** xref:caching-in-a-custom-policy-mule-4.adoc[Caching in a Custom Policy for Mule 4]
- ** xref:policies-custom-offline-landing-page.adoc[Offline Custom Policies]
-  *** xref:offline-policy-task.adoc[Applying Offline Custom Policies]
-  *** xref:offline-remove-task.adoc[Removing Offline Custom Policies]
+ ** xref:policies-mule4.adoc[Policies in Mule 4]
+  *** xref:available-policies.adoc[Categories]
+  *** xref:using-policies.adoc[Applying a Policy]
+  *** xref:re-order-policies-task.adoc[Re-ordering Policies]
+  *** xref:tutorial-manage-an-api.adoc[Applying a Policy and SLA Tier]
+  *** xref:resource-level-policies-about.adoc[Resource Level Policies]
+  *** xref:policy-scope-size-concept.adoc[Policy Packaging, Scope, and Size]
+  *** xref:disable-edit-remove-task.adoc[Disabling, Editing, or Removing a Policy]
+  *** xref:prepare-raml-task.adoc[Traits & policies concepts of RAML based APIs]
+  *** xref:policies-ootb-landing-page.adoc[Provided Policies]
+   **** xref:basic-authentication-simple-concept.adoc[Basic Authentication: Simple (Mule 4)]
```

Comment:
All policies under the Policies in Mule 4 section should not have the extra information "(Mule 4)".

Reply:
Agreed. I removed it here: https://github.com/mulesoft/docs-api-manager/pull/20/commits/4db4bcba76692610c182d772ceb2e6f7a69e8694

Reply:
Ok for the TOC. What about doc titles themselves?